### PR TITLE
Fix graph replication

### PR DESCRIPTION
**Pull request by safchain**

### CODE REVIEW

**Comment by safchain** [comment]
run skydive-k8s-tests

**Comment by safchain** [comment]
run skydive-functional-tests-backend-orientdb

**Comment by safchain** [comment]
run skydive-functional-tests-backend-orientdb

**Comment by safchain** [comment]
run skydive-functional-tests-backend-orientdb

File: graffiti/graph/orientdb.go [hunk 1]
```diff
@@ -319,7 +322,44 @@ func (o *OrientDBBackend) IsHistorySupported() bool {
...
+	}
+}
+
+func newOrientDBBackend(client orientdb.ClientInterface, electionService common.MasterElectionService) (*OrientDBBackend, error) {
+	o := &OrientDBBackend{
+		client: client,
+	}
+
+	if electionService != nil {
+		o.election = electionService.NewElection("es-graph-flush")
```
Comment (lebauce) [inline graffiti/graph/orientdb.go]: nit: orientdb-graph-flush

File: storage/orientdb/client.go [hunk 2]
```diff
@@ -561,8 +578,20 @@ func (c *Client) Query(obj string, query *filters.SearchQuery) (*Result, error)
 	return c.SQL(sql)
 }
 
-// Connect to the OrientDB server
-func (c *Client) Connect() error {
+// DeleteByQuery the OrientDB based on filters
+func (c *Client) DeleteByQuery(obj string, query *filters.SearchQuery) error {
```
Comment (lebauce) [inline storage/orientdb/client.go]: This function doesn't seem to be used

**Comment by safchain** [comment]
run skydive-functional-tests-backend-orientdb

**Pull request merged**